tests/test_D2_model_constraints.py

```diff
@@ -249,7 +253,6 @@ def test_add_constraints_minimal_renewable_share(self):
     pass
     """
 
-    """
     def test_add_constraints_minimal_renewable_share_None(self):
```

Comment:
Sorry, here was a mistake in the test function name: as far as I know the renewable share will never be None, but in [0,1]. When the minimal_renewable_share is 0, no constraint is added, if it is > 0, a constraint is added. So I would rename this function to `test_add_constraints_minimal_renewable_share_is_0()`.